invenio_search_ui/static/templates/invenio_search_ui/range.html

```diff
@@ -0,0 +1,9 @@
+<div class="panel panel-default" ng-show="vm.invenioSearchResults.aggregations.years.buckets.length > 0">
+    <div class="panel-heading">
+        <h3 class="panel-title">year</h3>
+    </div>
+    <div class="panel-body">
+        <div style="width: 220px; margin: 0 auto;" id="year_hist"></div>
+        <div style="width: 220px; margin: 0 auto;" id="year_select"></div>
+    </div>
+</div>
```

Comment:
I found no option for the `\0`, and besides adding it manually makes the file show up as a binary file on Github. Should I leave it in anyway?